blockchain/message_pool/src/msgpool/test_provider.rs

```diff
@@ -148,8 +148,14 @@ impl Provider for TestApi {
             }
             sequence += 1;
         }
-        let actor = ActorState::new(Cid::default(), Cid::default(), balance, sequence);
-        Ok(actor)
+        let actor = <ActorState as forest_shim::state_tree::Derive>::FVM::new(
```

Comment:
`Derive` sounds conflicting with macro derive thing, maybe we can name the trait as: 
`trait As {
    type Type;
}`

Reply:
This would look conflicting with both `as` and `type` keywords. I can change it to `Inner`; I have difficulty creating a meaningful name for this dummy trait. Does it sound good to you, @creativcoder?

Reply:
Right :) Yeah, Inner sounds fine